ckanext/ontario_theme/templates/internal/scheming/package/resource_read.html

```diff
@@ -15,221 +15,233 @@
-          {% set loading_text = _('Loading...') %}
-          {% set api_info_url = h.url_for(controller='api', action='snippet', ver=1, snippet_path='api_info.html', resource_id=res.id) %}
-          <ul>
-            <li><a class="" href="{{ api_info_url }}" data-module="api-info" data-module-template="{{ api_info_url }}" data-loading-text="{{ loading_text }}">{{ _('CKAN Data API calls') }}</a></li>
-          </ul>
-        </div>
+          {% endblock %}
+          <div class="action-group ontario-show-for-large">
+            <h2 class="ontario-h4">{{ _('Visualize data') }}</h2>
+            <p>{{ _('Use the Data Visualizer below to display this dataset as a table, graph or map.') }}</p>
+            <img src="/images/visualize_data.PNG" alt="" width="284" height="133"/>
+          </div>
+        {% endif %}
+      {% endblock %}
+      </div>
         {% endblock %}
-        <div class="action-group ontario-show-for-large">
-          <h2 class="ontario-h4">{{ _('Visualize data') }}</h2>
-          <p>{{ _('Use the Data Visualizer below to display this dataset as a table, graph or map.') }}</p>
-          <img src="/images/visualize_data.PNG" alt="" width="284" height="133"/>
         </div>
-      {% endif %}
-    {% endblock %}
-    </div>
       {% endblock %}
+      <div class="col-xs-12">
+        {% block data_preview %}
+          {% if res.format == "CSV" %}
+            {% block resource_view_nav %}
+              {% set resource_preview = h.resource_preview(resource, package) %}
+              {% snippet "package/snippets/resource_views_list.html",
+                          views=resource_views,
+                          pkg=pkg,
+                          is_edit=false,
+                          view_id=current_resource_view['id'],
+                          resource_preview=resource_preview,
+                          resource=resource
+              %}
+            {% endblock %}
+            {% block resource_view_content %}
+              {{ super() }}
+            {% endblock %}
+          {% endif %}
+        {% endblock %}
       </div>
-    {% endblock %}
-    <div class="col-xs-12">
-      {% block data_preview %}
-        {% if res.format == "CSV" %}
-          {% block resource_view_nav %}
-            {% set resource_preview = h.resource_preview(resource, package) %}
-            {% snippet "package/snippets/resource_views_list.html",
-                        views=resource_views,
-                        pkg=pkg,
-                        is_edit=false,
-                        view_id=current_resource_view['id'],
-                        resource_preview=resource_preview,
-                        resource=resource
-            %}
-          {% endblock %}
-          {% block resource_view_content %}
-            {{ super() }}
-          {% endblock %}
-        {% endif %}
-      {% endblock %}
-    </div>
-  </section>
+    </section>
+  {% else %}
+    <h2>{{ _('Resource not available') }}</h2>
```

Comment:
```suggestion
    <h1 class="col-xs-12">{{ _('Resource not available') }}</h1>
```
Since it's the only heading in the page it should be an `<h1>` and the added class is to align the left margin on all screens.

Reply:
i will make this and the translation change.